stl/inc/format

```diff
@@ -366,58 +366,363 @@ private:
+        } else if constexpr (is_same_v<_Ty, _CharType>) {
+            _Store_impl<_CharType>(_Arg_index, _Basic_format_arg_type::_Char_type, _Val);
+        } else if constexpr (is_same_v<_Ty, char> && is_same_v<_CharType, wchar_t>) {
+            _Store_impl<_CharType>(_Arg_index, _Basic_format_arg_type::_Char_type, static_cast<wchar_t>(_Val));
+        } else if constexpr (signed_integral<_Ty> && sizeof(_Ty) <= sizeof(int)) {
+            _Store_impl<int>(_Arg_index, _Basic_format_arg_type::_Int_type, static_cast<int>(_Val));
+        } else if constexpr (unsigned_integral<_Ty> && sizeof(_Ty) <= sizeof(unsigned int)) {
+            _Store_impl<unsigned int>(_Arg_index, _Basic_format_arg_type::_UInt_type, static_cast<unsigned int>(_Val));
+        } else if constexpr (signed_integral<_Ty> && sizeof(_Ty) <= sizeof(long long)) {
+            _Store_impl<long long>(_Arg_index, _Basic_format_arg_type::_Long_long_type, static_cast<long long>(_Val));
+        } else if constexpr (unsigned_integral<_Ty> && sizeof(_Ty) <= sizeof(unsigned long long)) {
+            _Store_impl<unsigned long long>(
+                _Arg_index, _Basic_format_arg_type::_ULong_long_type, static_cast<unsigned long long>(_Val));
+        } else if constexpr (is_same_v<_Ty, float>) {
+            _Store_impl<float>(_Arg_index, _Basic_format_arg_type::_Float_type, _Val);
+        } else if constexpr (is_same_v<_Ty, double>) {
+            _Store_impl<double>(_Arg_index, _Basic_format_arg_type::_Double_type, _Val);
+        } else if constexpr (is_same_v<_Ty, long double>) {
+            _Store_impl<long double>(_Arg_index, _Basic_format_arg_type::_Long_double_type, _Val);
+        } else {
+            _STL_VERIFY(false, "Invalid type passed to _Format_arg_store::_Store");
+        }
+    }
 
-    using _Value_type = basic_format_arg<_Context>;
+    void _Store(const size_t _Arg_index, const _CharType* _Val) noexcept {
+        _Store_impl<const _CharType*>(_Arg_index, _Basic_format_arg_type::_CString_type, _Val);
+    }
+
+    template <class _Traits>
+    void _Store(const size_t _Arg_index, basic_string_view<_CharType, _Traits> _Val) noexcept {
+        _Store_impl<basic_string_view<_CharType>>(
+            _Arg_index, _Basic_format_arg_type::_String_type, basic_string_view<_CharType>{_Val});
+    }
+
+    template <class _Traits, class _Alloc>
+    void _Store(const size_t _Arg_index, const basic_string<_CharType, _Traits, _Alloc>& _Val) noexcept {
+        _Store_impl<basic_string_view<_CharType>>(
+            _Arg_index, _Basic_format_arg_type::_String_type, basic_string_view<_CharType>{_Val.data(), _Val.size()});
+    }
+
+    void _Store(const size_t _Arg_index, nullptr_t) noexcept {
+        _Store_impl<const void*>(_Arg_index, _Basic_format_arg_type::_Pointer_type, static_cast<const void*>(nullptr));
+    }
+
+    // clang-format off
+    template <class _Ty>
+        requires is_void_v<_Ty>
+    void _Store(const size_t _Arg_index, const _Ty* p) noexcept {

```

Comment:
`p` is still non-`_Ugly` (the previous comment was resolved but I don't see a change).

Reply:
What, I am 100% sure that I changed them... 

Maybe I resetted something and did not add back? Wish git would really do what I want not what I do